build/generated.bzl

```diff
@@ -0,0 +1,149 @@
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# The go_prefix for this repository, needed to map relative package paths into
+# a full-resolved go package.
+go_prefix = "k8s.io/kubernetes"
+
+# Pretty-prints the provided dict -> dict -> dict as a dict -> dict -> list,
+# appending lines into output.
+def _format_output_dict(id, tags, output):
```

Comment:
Can we unit test this method?

Reply:
maybe. there is a `unittest.bzl` module in bazel-skylib, though I haven't tried using it yet.

Reply:
the openapi build rule is also its own sort of unit test of this function - if bazel fails to parse `tags.bzl`, then it's probably a bug here. :)